### PR TITLE
[EH][DDCE-1511] Bootstrap 5.3.0 and Play 2.7

### PR DESCRIPTION
**Pull request by LetMeTestThat**
# DDCE-1511

- Bootstrap 2.3.0 -> 5.3.0
- Play 2.6.25 -> 2.7.9
- Sbt 1.3.13 -> 1.4.9
- General depedency bumps
- Bunch of unit test pain

 - [x]  I've made every effort to commit high quality, clean code and I have executed relevant static analyses to be sure
 - [x]  I've included appropriate tests with any code I've added (Unit, Integration, Acceptance etc.)
 - [x]  I've executed the acceptance test pack locally to ensure there are no functional regressions (Performance tests where relevant)
 - [x]  I've squashed my commits - including the JIRA issue number in the commit message
 - [x]  I've run a dependency check to ensure all dependencies are up to date

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,15 +1,13 @@
...
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.25")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.9")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-settings" % "4.7.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-settings" % "4.8.0")
```
Comment (DannyPennington97) [inline project/plugins.sbt:10]: sbt-settings is pulled in by auto-build so can be removed

File: test/controllers/ReportableEventsControllerSpec.scala [hunk 2]
```diff
@@ -113,16 +114,26 @@ class ReportableEventsControllerSpec extends UnitSpec with ERSFakeApplicationCon
     "direct to ers errors page if fetching ersMetaData throws exception" in {
       val controllerUnderTest = buildFakeReportableEventsController(ersMetaDataRes = false)
       val result = controllerUnderTest.updateErsMetaData(ersRequestObject)(Fixtures.buildFakeUser, Fixtures.buildFakeRequestWithSessionIdCSOP("GET"), hc)
+      result match {
```
Comment (DannyPennington97) [inline test/controllers/ReportableEventsControllerSpec.scala:117]: Is the match necessary here (and below)? I feel like just having lines 119 & 120 does the same thing

Comment (DannyPennington97) [reply test/controllers/ReportableEventsControllerSpec.scala:117]:
```val result = controllerUnderTest.updateErsMetaData(ersRequestObject)(Fixtures.buildFakeUser, Fixtures.buildFakeRequestWithSessionIdCSOP("GET"), hc).asInstanceOf[Future[Result]]
      status(result) shouldBe 500
      contentAsString(result) shouldBe contentAsString(Future.successful(controllerUnderTest.getGlobalErrorPage))
    }
    ```

Comment (LetMeTestThat) [reply test/controllers/ReportableEventsControllerSpec.scala:117]: As discussed, I'll change this from the pattern match and call `asInstanceOf` instead

**Comment by DannyPennington97** [comment]
Everything else looking good, once above changes are in this is ready to merge

**Review by DannyPennington97 — APPROVED** [review]

**Pull request merged**